examples/Example4_OLED_Distance/Example4_OLED_Distance.ino

```diff
@@ -0,0 +1,100 @@
+/* SparkFun Ulrasonic Distance Sensor - Example 4 Basic Distance Sensing on an OLED Display
+ * 
+ * Products: 
+ *  *  SparkFun Qwiic Ultrasonic Distance Sensor - HC-SR04 (SEN-1XXXX)
+ *  *  https://www.sparkfun.com/1XXXX
+ *  *  SparkFun Qwiic Narrow OLED Display (LCD-1XXXX)
+ *  *  https://www.sparkfun.com/1XXXX
+ * 
+ * Written By: Elias Santistevan
+ * Date: 06/2024
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * Copyright (c) 2024 SparkFun Electronics
+ */
+
+#include "SparkFun_Qwiic_OLED.h"
```

Comment:
Add a link to the OLED library for users who don't already have it

Reply:
Added